src/BlazorWebView/src/Wpf/BlazorWebView.cs

```diff
@@ -129,6 +139,16 @@ public string HostPage
 			set => SetValue(HostPageProperty, value);
 		}
 
+		/// <summary>
+		/// Path for initial Blazor navigation when the Blazor component is finished loading. For example, "/".
+		/// This property must be set to a valid value for the Razor components to start.
```

Comment:
This extra comment line is here but not on the other properties. I think they should all be consistent, and if you agree, it's up to you to decide how to unify them.